### ISSUE TITLE
Not able to read ensemble output "biomod.ensemble.model.out" using load function.

### ISSUE THREAD

**Issue by prajapatijyoti23** [biomodhub/biomod2#388]
_Please make sure to close the issue once you consider it as solved_
_Please use screenshots only when you cannot copy-paste the object, e.g. for figures or maps_

**Error and context**
I ran all my codes in 2020. The outputs of the codes are of 2020. I could run the codes and get the values from the model's output files during that year. Today, I am trying to run the same codes to get values from the output files. I am getting errors.

**Code used to get the error**
pastee ="Lagascea.mollis"
dir1 <- paste("I:/Bhowmick_sir/values_great20",paste(pastee, "VIF", sep="."), sep="/")
valu1 <- list.files(dir1, pattern=".out")
model.load1 <- load(paste(dir1,valu1[2], sep="/"))
model.load1
model.out1 <-get(model.load1)
model.eval1<-biomod2::get_evaluations(model.out1)
```
# Error with codes
<html>
<body>
<!--StartFragment-->
>     pastee ="Lagascea.mollis" 
>     dir1 <- paste("I:/Bhowmick_sir/values_great20",paste(pastee, "VIF", sep="."), sep="/") 
>     valu1 <- list.files(dir1, pattern=".out") 
>     model.load1 <- load(paste(dir1,valu1[2], sep="/"))
 >     model.load1 [1] "Lagascea.mollis.VIF.1611584068ensemble.models.out" 
Warning messages: 
1: Not a validObject(): no slot of name "dir_name" for this object of class "EMmean_biomod2_model"  
2: Not a validObject(): no slot of name "dir_name" for this object of class "EMcv_biomod2_model"  
3: Not a validObject(): no slot of name "thresholds" for this object of class "EMca_biomod2_model"  
4: Not a validObject(): no slot of name "dir_name" for this object of class "EMwmean_biomod2_model"  
5: Not a validObject(): no slot of name "dir_name" for this object of class "EMmean_biomod2_model"  
6: Not a validObject(): no slot of name "dir_name" for this object of class "EMcv_biomod2_model"  
7: Not a validObject(): no slot of name "thresholds" for this object of class "EMca_biomod2_model"  
8: Not a validObject(): no slot of name "dir_name" for this object of class "EMwmean_biomod2_model"  
>     model.out1 <-get(model.load1) 
>     model.eval1<-biomod2::get_evaluations(model.out1) 
Error in (function (classes, fdef, mtable)  :    unable to find an inherited method for function ‘get_evaluations’ for signature ‘"BIOMOD.EnsembleModeling.out"’ 
In addition: Warning messages: 
1: Not a validObject(): no slot of name "dir_name" for this object of class "EMmean_biomod2_model"  
2: Not a validObject(): no slot of name "dir_name" for this object of class "EMcv_biomod2_model"  
3: Not a validObject(): no slot of name "thresholds" for this object of class "EMca_biomod2_model"  
4: Not a validObject(): no slot of name "dir_name" for this object of class "EMwmean_biomod2_model"  
5: Not a validObject(): no slot of name "dir_name" for this object of class "EMmean_biomod2_model"  
6: Not a validObject(): no slot of name "dir_name" for this object of class "EMcv_biomod2_model"  
7: Not a validObject(): no slot of name "thresholds" for this object of class "EMca_biomod2_model"  
8: Not a validObject(): no slot of name "dir_name" for this object of class "EMwmean_biomod2_model"
--
 
> | >
>

<!--EndFragment-->
</body>
</html>```

**Environment Information**
<html>
<body>
<!--StartFragment-->
> version  _                                 
platform       x86_64-w64-mingw32                
arch           x86_64                            
os             mingw32                           
crt            ucrt                              
system         x86_64, mingw32                   
status                                           
major          4                                 
minor          3.0                               
year           2023                              
month          04                                
day            21                               
svn rev        84292                            
language       R                                 
version.string R version 4.3.0 (2023-04-21 ucrt) 
nickname       Already Tomorrow
--
 
> | >
>

<!--EndFragment-->
</body>
</html>```
# paste output of sessionInfo() here
```

**Additional information**
If you have any additional information or context you can add it here.

**Comment by prajapatijyoti23** [comment]
[values.zip](https://github.com/biomodhub/biomod2/files/13765811/values.zip)

Here is the output files that I generated in year 2020.

Thank you.

**Comment by prajapatijyoti23** [comment]
Hello, I got the solution. I can read the output files on the R version of 3.6.3. I downloaded the version built in 2020 from https://cran.r-project.org/bin/windows/base/old/3.6.3/ and biomod package, I downloaded tar file of 2020 from https://cran.r-project.org/src/contrib/Archive/biomod2/. 
So yes, now I can read my model output files of biomod2.

Here are the following codes.

> library(biomod2)
> pastee ="Lagascea.mollis"
> dir1 <- paste("G:/Bhowmick_sir/values_great20",paste(pastee, "VIF", sep="."), sep="/")
> valu1 <- list.files(dir1, pattern=".out")
> model.load1 <- load(paste(dir1,valu1[2], sep="/"))
> model.load1
[1] "Lagascea.mollis.VIF.1611584068ensemble.models.out"
> model.out1 <-get(model.load1)
> model.eval1<-biomod2::get_evaluations(model.out1)
> 
> model.eval1
$Lagascea.mollis.VIF_EMmeanByTSS_mergedAlgo_mergedRun_mergedData
    Testing.data Cutoff Sensitivity Specificity
TSS        0.855  484.0       96.61      88.922
ROC        0.979  491.5       96.61      89.222

Thank you once again.
